features/environment.py

```
@@ -8,3 +9,21 @@ def before_all(context: Context) -> None:
 
     allure_report("allure-results")
+
+
+TEMP_DIRS = ["/tmp/sandbox/behave"]
+
+def clean_temp_dirs():
+    """Remove old test directories before running tests."""
+    for temp_dir in TEMP_DIRS:
+        os.system(f"rm -rf {temp_dir}")
+
+def before_all(context):
```

Ruff (0.8.2)

Redefinition of unused `before_all` from line 8

(F811)